src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js

```diff
@@ -0,0 +1,43 @@
+import path from 'ramda/src/path';
+import { buildATIPageTrackPath } from '../../atiUrl';
+import { getPublishedDatetime } from '../../../../lib/analyticsUtils';
+
+export const buildCPSATIParams = (pageData, requestContext, serviceContext) => {
```

Comment:
Is there a reason we are passing `requestContext` and `serviceContext` as props rather than `useContext` here?

Reply:
Just the way the architecture is set up:  https://github.com/bbc/simorgh/blob/latest/src/app/containers/ATIAnalytics/params/index.js#L35-L41

All the page types are doing this